client/keys/show.go

```diff
@@ -18,30 +22,67 @@ const (
 	FlagPublicKey = "pubkey"
 	// FlagBechPrefix defines a desired Bech32 prefix encoding for a key.
 	FlagBechPrefix = "bech"
+
+	flagMultiSigThreshold = "multisig-threshold"
 )
 
+var _ keys.Info = (keys.Info)(nil)
+
+type multiSigKey struct {
```

Comment:
Will `multiSigKey` be used elsewhere, besides just via `show`? If so, may I suggest to moving it to it's own file, even though it's fairly small.

Reply:
It's presently not used elsewhere.